app/cafe.py

```diff
@@ -0,0 +1,29 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        name = visitor["name"]
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{name} is not vaccinated")
+
+        vaccine = visitor["vaccine"]
+        expiration_date = vaccine["expiration_date"]
+
+        if expiration_date < date.today():
+            raise OutdatedVaccineError(f"{name}'s vaccine is expired")
```

Comment:
if the expiration_date key is not present, the program will terminate its execution (Aishka is complaining about this in this task, so it will still need to be fixed)
```suggestion
        expiration_date = vaccine.get("expiration_date", date(2000, 1, 1)) # if there is no date, then we take the default as knowingly overdue

        if expiration_date < date.today():
            raise OutdatedVaccineError(f"{name}'s vaccine is expired")
```